src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -1898,6 +1774,146 @@ public XMPPError onRemoveSource(JingleSession sourceJingleSession,
             sourceJingleSession, sourcesToRemove, sourceGroupsToRemove, true);
     }
 
+    /**
+     * Updates the RTP description, transport and propagates sources and source
+     * groups of a participant that sends the session-accept or transport-accept
+     * Jingle IQs.
+     *
+     * @param jingleSession
+     * @param contents
+     * @return
+     */
+    private XMPPError updateParticipant(
+        JingleSession jingleSession,
+        List<ContentPacketExtension> contents)
+    {
+        Participant participant
+            = findParticipantForJingleSession(jingleSession);
+        Jid participantJid = jingleSession.getAddress();
+
+        if (participant == null)
+        {
+            String errorMsg
+                = "No participant found for: " + participantJid;
+            logger.warn(errorMsg);
+            return XMPPError.from(XMPPError.Condition.item_not_found,
+                errorMsg).build();
+        }
+
+        // XXX We will be acting on the received session-accept bellow.
```

Comment:
This comment is now out-of-date since this is not just used for session-accept.  Also, spelling: "bellow" should be "below".

Reply:
Is it out-of-date? It was written back in 2015 in c975f1a and it seems the dance is still present in the code (if you look at another comment in today's code [here](https://github.com/jitsi/jicofo/blob/2696bb4c901682b8fcb8670fccc75436634a2c55/src/main/java/org/jitsi/protocol/xmpp/AbstractOperationSetJingle.java#L229)).

Reply:
I meant that it was saying "on the received session-accept" whereas this now applies to both session-accept and transport-accept.

Reply:
(But you fixed it.)